rt/transports/http-hc/src/main/java/org/apache/cxf/transport/http/asyncclient/AsyncHTTPConduit.java

```diff
@@ -561,7 +561,10 @@ protected void verifySession(final HttpHost host,
                                     setSSLSession(sslsession);
                                 }
                             });
+                    // See please https://issues.apache.org/jira/browse/HTTPASYNC-168, the attribute names
+                    // are case-sensitive.
                     ctx.setAttribute("http.iosession-factory-registry", regBuilder.build());
```

Comment:
@ffang may I ask you to take a look please? thank you!

Reply:
Ah, that's it, thanks!